contrib/ansible/roles/skydive_common/tasks/package.yml

```diff
@@ -1,12 +1,13 @@
 ---
-- name: Upgrade all packages
-  yum: name=* state=latest
-  when: ansible_os_family == 'RedHat'
+#- name: Upgrade all packages
```

Comment:
I'm quite OK with this patch, can you simply remove the lines instead of comments ?